ethdb/privateapi/ethbackend.go

```diff
@@ -18,6 +18,7 @@ import (
 // EthBackendAPIVersion
 // 2.0.0 - move all mining-related methods to 'txpool/mining' server
 // 2.1.0 - add NetPeerCount function
+// 2.2.0 - add NodesInfo function
 var EthBackendAPIVersion = &types2.VersionReply{Major: 2, Minor: 1, Patch: 0}
```

Comment:
need increase version?

Reply:
Hmm, yes, I wonder why this did not come in... Will check, thank you

Reply:
Oh, I see I think @dsavelev bumped the version in the comment but forgot to bump it in the variable. We will need to make PR for this into `devel` and cherry-pick to here

Reply:
I will bump to 3.0.0 in `devel` and to 2.2.0 in this PR (`stable`)

Reply:
hm, that's strange, I'm pretty sure I did it. Sorry about that